### PR TITLE
feat: adjust etna builder props

### PR DESCRIPTION
**Pull request by erictaylor**
This pull request includes several changes to the `src/vms/pvm/etna-builder/builder.ts` file and its related test files. The primary focus is on refactoring the code to replace the usage of `SpendOptions` with individual parameters and improving code readability. Additionally, a minor typo was corrected in `src/vms/common/models.ts`.

## Changes
1. Removal of `spendOptions` prop.
2. Add `changeAddressesBytes` common builder prop. Defaults to use the `fromAddressesBytes`.
3. Add `memo` common builder prop. Defaults to empty Uint8Array.
4. Add `minIssuanceTime` common builder prop. Defaults to current unix time in seconds.
5. Renamed `newImportTx` `toAddresses` prop to `toAddressesBytes`.

## Migration
* `spendOptions.changeAddresses` -> `changeAddressesBytes`.
* `spendOptions.memo` -> `memo`.
* `spendOptions.minIssuanceTime` -> `minIssuanceTime`.
* On import tx's `toAddresses` -> `toAddressesBytes`.

Any usage of `spendOptions.locktime` or `spendOptions.threshold` need removed. Those props didn't do anything in the builder, and therefore caused confusion.

### CODE REVIEW

File: src/vms/pvm/etna-builder/spend.ts [hunk 1]
```diff
@@ -37,6 +47,20 @@ type SpendResult = Readonly<{
...
+   */
+  changeAddressesBytes?: readonly Uint8Array[];
+  /**
+   * Optionally specifies the output owners to use for the unlocked
+   * AVAX change output if no additional AVAX was needed to be burned.
+   * If this value is `undefined` or `null`, the default change owner is used.
+   *
+   * Used in ImportTx.
+   */
+  changeOwnerOverride?: OutputOwners | null;
```
Comment (rictorlome) [inline src/vms/pvm/etna-builder/spend.ts:63]: Can `changeOwnerOverride` and `changeAddressesBytes` be consolidated? Not sure why you need to specify change owners via 2 parameters. Is having 2 parameters an API oversight we are inheriting or is there a legitimate usecase for over specifying 2 different change owners and having `changeAddressesBytes` overridden?

**Review by rictorlome — COMMENTED** [review]
I am good with this for now. I think, per our conversation with @bferenc , we might want to actually make use of the threshold and locktime, but that can be a follow up.
Also left a question regarding duplication of changeOutput parameters.

File: src/vms/pvm/etna-builder/spend.ts (hunk 1 above)
Comment (erictaylor) [reply src/vms/pvm/etna-builder/spend.ts]: Changes in e0835e4 should resolve.

**Review by rictorlome — APPROVED** [review]
lgtm

**Pull request merged**